scss/tech.scss

```diff
@@ -1,3 +1,37 @@
+@import 'colors';
+@import 'fonts';
+
+.tech-container {
```

Comment:
This is the background border right? This shouldnt be the main class, the tech container should be the one with green backgound and then use a pseudo-element for the external border